lib/modules/manager/bun-version/index.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { extractPackageFile } from '.';
+
+describe('modules/manager/bun-version/index', () => {
+  describe('extractPackageFile()', () => {
+    it('returns a result', () => {
+      const res = extractPackageFile('1.1.15\n');
+      expect(res.deps).toEqual([
+        {
+          depName: 'bun',
+          packageName: 'oven-sh/bun',
+          commitMessageTopic: 'Bun',
+          currentValue: '1.1.15',
+          datasource: 'github-releases',
+          extractVersion: '^bun-v(?<version>\\S+)',
+          versioning: 'semver',
+        },
+      ]);
+    });
+
+    it('handles ranges', () => {
+      const res = extractPackageFile('1.0\n');
+      expect(res.deps).toEqual([
+        {
+          depName: 'bun',
+          packageName: 'oven-sh/bun',
+          commitMessageTopic: 'Bun',
+          currentValue: '1.0',
```

Comment:
That would not work with `semver` versioning as it only contains two digits. Use `semver-coerced` instead.